### PR TITLE
Move creative pagination buttons

### PR DESCRIPTION
**Pull request by Geometrically**
A fix for #336 to make it work with Fabric's latest version. Here's a GIF of it in action:

Sorry for the 2 line PR!

### CODE REVIEW

**Comment by shedaniel** [comment]
you still need to change the texture, that was the main part

File: fabric-item-groups-v0/src/main/java/net/fabricmc/fabric/mixin/item/group/client/MixinCreativePlayerInventoryGui.java [hunk 1]
```diff
@@ -120,8 +120,8 @@ private void fabric_updateSelection() {
 	private void init(CallbackInfo info) {
 		fabric_updateSelection();
 
-		int xpos = x + 170;
-		int ypos = y + 4;
+		int xpos = x + 116;
+		int ypos = y - 11;
```
Comment (Ueaj-Kerman) [inline fabric-item-groups-v0/src/main/java/net/fabricmc/fabric/mixin/item/group/client/MixinCreativePlayerInventoryGui.java]: magic number, needs docs

**Review by Ueaj-Kerman — APPROVED** [review]

**Comment by Geometrically** [comment]
Didn't see that part, will do.

File: fabric-item-groups-v0/src/main/java/net/fabricmc/fabric/mixin/item/group/client/MixinCreativePlayerInventoryGui.java (hunk 1 above)
Comment (modmuss50) [reply fabric-item-groups-v0/src/main/java/net/fabricmc/fabric/mixin/item/group/client/MixinCreativePlayerInventoryGui.java]: No. this is fine. please dont meme.

**Comment by shedaniel** [comment]
also remember to bump the module version, and change the buttons dimensions to fit the new texture

**Comment by Geometrically** [comment]
Done. Here's the updated texture:
Screenshot_20200524_131259

**Comment by TheBrokenRail** [comment]
Does it show the hover texture?

File: fabric-item-groups-v0/build.gradle [hunk 2]
```diff
@@ -1,5 +1,5 @@
 archivesBaseName = "fabric-item-groups-v0"
-version = getSubprojectVersion(project, "0.1.12")
+version = getSubprojectVersion(project, "0.1.13")
```
Comment (modmuss50) [inline fabric-item-groups-v0/build.gradle]: Id almost go to 0.2.0 no reason not to tbh.

Comment (Geometrically) [reply fabric-item-groups-v0/build.gradle]: true, I'll bump to that and remove the krita generated file that got left in for some reason

**Comment by Geometrically** [comment]
It does show the tooltip if that is what you meant:
Screenshot_20200524_132755

**Comment by TheBrokenRail** [comment]
What about the blue highlight/hover texture though?

**Comment by shedaniel** [comment]
I would like to have the texture in #336 tbh.

**Comment by Geometrically** [comment]
That texture is lost, I'll add a hover texture though

**Comment by shedaniel** [comment]
it is not lost. it is in the files changed list.

**Comment by Geometrically** [comment]
Fixed. Here's the final gif:

**Comment by shedaniel** [comment]
Texture in this PR:
image

This texture is still different than the one in the original PR, and I would prefer the texture in the original PR much more imo. But this is just my preference, you are free to disagree.

**Comment by Geometrically** [comment]
It's the same texture. I took the file from the original PR

**Review by Prospector — APPROVED** [review]

**Review by shedaniel — APPROVED** [review]

**Comment by modmuss50** [comment]
Please fix checkstyle.

**Comment by Geometrically** [comment]
Fixed, turns out it was just a tiny whitespace

**Review by liach — APPROVED** [review]

**Review by Hugman76 — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Review by Player3324 — APPROVED** [review]

**Review by haykam821 — APPROVED** [review]

**Review by hYdos — APPROVED** [review]

**Comment by ExtraCrafTX** [comment]
Sorry about the abandoned PR, and yes, the texture here is indeed slightly different. Well, the texture itself may be the same, but it is misplaced by 1 pixel. You can see that there is a one-pixel gap to the left and a 2-pixel gap to the right.

**Comment by Prospector** [comment]
There were commits after the last screenshot so I assume it was fixed?

**Comment by Geometrically** [comment]
Yep, it was fixed.

**Pull request merged**